com.gitlab.zehkira.Myuzi.yml

```
@@ -0,0 +1,21 @@
+app-id: com.gitlab.zehkira.Myuzi
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: myuzi
+modules:
+  - python3-modules.json
+  - name: myuzi
+    buildsystem: simple
+    build-commands:
+      - make install-flatpak DESTDIR=/app
+    sources:
+      - type: git
+        url: https://gitlab.com/zehkira/myuzi.git
+        tag: v1.0.0
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
```

```suggestion
  - --socket=x11
  - --share=ipc
```

This is required for X11.

Also this doesn't support Wayland?

---

I guess it does support Wayland. Is `share=ipc` still required with `socket=wayland` + `socket=fallback-x11`?